featuretools/entityset/entityset.py

```diff
@@ -909,19 +909,65 @@ def add_last_time_indexes(self, updated_entities=None):
     #  Other ###############################################
     ###########################################################################
 
-    def add_interesting_values(self, max_values=5, verbose=False):
+    def add_interesting_values(self, max_values=5, verbose=False, datatable_id=None):
         """Find interesting values for categorical variables, to be used to generate "where" clauses
 
         Args:
             max_values (int) : Maximum number of values per variable to add.
             verbose (bool) : If True, print summary of interesting values found.
+            datatable_id (str) : The DataTable for which to add interesting values. If not specified
```

Comment:
Maybe this should be a higher level discussion, but should this be `datatable_id` or `entity_id`?  Are we still keeping entities as a concept, and if so when should we refer to a thing as an entity or the datatable?

Reply:
I've gone back and forth on this. Maybe the safe thing for now is to leave it as `entity_id` and if the concept of an `Entity` goes away we can rename to something more appropriate at that time.

Reply:
Keep `entity_id` as the parameter to maintain status quo? We did this in the other MR as well: 
- https://github.com/alteryx/featuretools/pull/1276#discussion_r541235051

Reply:
Updated to use `entity_id` here.